### DISCUSSION TITLE
SCTP Multihoming support

### DISCUSSION THREAD

**Discussion by drz6** [loxilb-io/loxilb#95] [Q&A] [answered]
Hi All!

Does LoxiLB support SCTP multihoming? Not much about this in the docs.

Thanks!

**Comment by UltraInstinct14** [comment, marked as answer]
Currently SCTP multihoming is not supported. But we have plans to support it down the lane.

**Reply by UltraInstinct14** [reply]
We are glad to announce the availability of multi-homing [feature](https://www.loxilb.io/post/k8s-introducing-sctp-multihoming-functionality-with-loxilb)

Hope OP finds it useful !!
